2020-12-21 22:47:02,622 INFO    MainThread:25807 [wandb_setup.py:_flush():70] setting env: {}
2020-12-21 22:47:02,622 INFO    MainThread:25807 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2020-12-21 22:47:02,623 INFO    MainThread:25807 [wandb_setup.py:_flush():70] setting login settings: {}
2020-12-21 22:47:02,623 INFO    MainThread:25807 [wandb_init.py:_log_setup():303] Logging user logs to /home/kim/torch_mimic/wandb/run-20201221_224702-1q3vdcth/logs/debug.log
2020-12-21 22:47:02,623 INFO    MainThread:25807 [wandb_init.py:_log_setup():304] Logging internal logs to /home/kim/torch_mimic/wandb/run-20201221_224702-1q3vdcth/logs/debug-internal.log
2020-12-21 22:47:03,972 INFO    MainThread:25807 [wandb_run.py:_console_start():1237] atexit reg
2020-12-21 22:47:03,972 INFO    MainThread:25807 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.REDIRECT
2020-12-21 22:47:03,973 INFO    MainThread:25807 [wandb_run.py:_redirect():1107] Redirecting console.
2020-12-21 22:47:03,973 INFO    MainThread:25807 [redirect.py:install():204] install start
2020-12-21 22:47:03,974 INFO    MainThread:25807 [redirect.py:install():219] install stop
2020-12-21 22:47:03,974 INFO    MainThread:25807 [redirect.py:install():204] install start
2020-12-21 22:47:03,974 INFO    MainThread:25807 [redirect.py:install():219] install stop
2020-12-21 22:47:03,974 INFO    MainThread:25807 [wandb_run.py:_redirect():1151] Redirects installed.
2020-12-22 10:48:01,243 ERROR   stdout    :25807 [redirect.py:_pipe_relay():130] problem in pipe relay
Traceback (most recent call last):
  File "/home/kim/anaconda3/envs/torch_rl/lib/python3.6/site-packages/wandb/sdk/lib/redirect.py", line 128, in _pipe_relay
    cb(name, data)
  File "/home/kim/anaconda3/envs/torch_rl/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 602, in _console_callback
    self._backend.interface.publish_output(name, data)
  File "/home/kim/anaconda3/envs/torch_rl/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 151, in publish_output
    self._publish_output(o)
  File "/home/kim/anaconda3/envs/torch_rl/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 156, in _publish_output
    self._publish(rec)
  File "/home/kim/anaconda3/envs/torch_rl/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 439, in _publish
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
